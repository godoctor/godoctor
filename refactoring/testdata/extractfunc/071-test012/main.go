//<<<<<extract,9,3,11,3,FOO,fail
package main

import "fmt"

func main() {
	i := 0
	for i <= 5 {
		if i == 3 {
			break
		}
		fmt.Println(i)
		i++
	}
	fmt.Println("after loop")
}
