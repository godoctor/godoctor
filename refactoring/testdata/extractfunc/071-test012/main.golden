//<<<<<extract,8,2,14,2,FOO,fail
package main

import "fmt"

func main() {
	i := 0
	FOO(i)
	fmt.Println("after loop")
}
func FOO(i int) {
	for i <= 5 {
		if i == 3 {
			break
		}
		fmt.Println(i)
		i++
	}
}
